CHANGELOG.md

```diff
@@ -0,0 +1,39 @@
+# Changelog
+
+
+## [0.2.1] - 2024-03-25
+
+### Changed
+
+- Update from `Mix.Config` to `Config` ([#4](https://github.com/elixir-waffle/waffle_gcs/pull/4)) (Rafael Scheffer, @rschef)
+- Bump `google_api_storage` from `0.29` to `0.34` ([#4](https://github.com/elixir-waffle/waffle_gcs/pull/4)) (Rafael Scheffer, @rschef)
+- Bump `waffle` from `1.1.5` to `1.1.8`
+- Bump `ex_doc` from `0.25` to `0.31`
+- Bump `dialyxir` from `1.1.0` to `1.4.3`
+
+
+### Removed
+
+- **Breaking:** drop maintenance support for Elixir versions `< v1.12`
```

Comment:
If this has happened, should the mix.exs be updated to reflect the changed version requirement?

Reply:
That's actually why I haven't merged this. I think I originally added this something it would go with the jose update, but I may just remove this?

I need to run it to confirm 1.9 would still work though.